tests/manual/media/FXMediaPlayer/src/fxmediaplayer/FXMediaPlayer.java

```diff
@@ -162,6 +162,7 @@ public synchronized void onSourceChanged(String source) {
         }
 
         // Create Media, MediaPlayer, MediaView and ImageView
+        System.out.println("FXMediaPlayer source: " + source);
```

Comment:
probably needs to be removed?

Reply:
No, I done it on purpose to see which files we loading. Makes it easy to test a lot of files via playlist.